cjwkernel/types.py

```diff
@@ -547,12 +555,18 @@ def TODO_i18n(cls, text: str) -> I18nMessage:
         return cls("TODO_i18n", {"text": text})
 
     @classmethod
-    def from_dict(cls, value: Dict[str, Any]) -> I18nMessage:
+    def from_dict(cls, value: I18nMessageDict) -> I18nMessage:
         return cls(value["id"], value["arguments"])
 
-    def to_dict(self) -> Dict[str, Any]:
+    def to_dict(self) -> I18nMessageDict:
         return {"id": self.id, "arguments": self.args}
 
+    def localize(self, locale_id: str) -> str:
```

Comment:
Nix this. We can't localize from within the `cjwkernel` package, and nobody should want to.